web/src/components/MalteHelpForm.tsx

```diff
@@ -15,10 +19,10 @@ const Note = styled.div`
   padding-bottom: 10px;
 `
 
-const StyledIcon = styled(Icon)`
+const StyledIcon = styled(Icon)<{ direction: 'ltr' | 'rtl' }>`
   width: 25px;
   height: 25px;
-  padding-right: 20px;
+  ${props => (props.direction === 'ltr' ? 'padding-right: 20px' : 'padding-left: 20px')};
```

Comment:
Could we instead use a gap on the parent component to avoid these direction dependent styles?

Reply:
Not sure what you mean...

Reply:
You can add `gap: 20px;` on the parent component (in this case `Note`) and remove this line with the same effect and without the need for a condition/pass the direction as a prop.